src/Constants_Conversion.f90

```diff
@@ -33,6 +33,7 @@ MODULE Constants_Conversion
   REAL(SRK),PUBLIC,PARAMETER ::    ZERO=0.000000000000000_SRK !16 digits
   REAL(SRK),PUBLIC,PARAMETER ::     ONE=1.000000000000000_SRK !16 digits
   REAL(SRK),PUBLIC,PARAMETER ::     TWO=2.000000000000000_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::    HALF=0.500000000000000_SRK !16 digits
```

Comment:
One other comment on this based on experience.

At one point I thought it'd be useful to put lots of common numbers like this in a module (I think I did this for TRIAD and DeCART) and after some time, I found it REALLY annoying to move code around have this module. It was easier to just type out ```0.0d0``` or ```0.0_SRK``` because this prevents a dependency. Normally though I'll go with approaches that require the least amount of typing.

Reply:
yeah, I go back and forth.  I needed Pi anyway, so it was already loaded.  I added HALF because it saves some room.  In nodal, I added a lot of local parameters for other numbers 4, 7, etc. but there wasn't a lot of reason to add those here since they are fairly uncommon.  I think common numbers here makes sense.